chooloolib/src/main/java/com/chooloo/www/chooloolib/ui/dialpad/DialpadFragment.kt

```diff
@@ -34,6 +34,9 @@ open class DialpadFragment @Inject constructor() : BaseFragment<DialpadViewState
                 addTextContextMenuItemListener(this@DialpadFragment)
             }
 
+            //promote edit to the viewstate
+            viewState.editTextRef.value = dialpadEditText
```

Comment:
Please remove any view references from the viewstate
The view should listen for changes and act accordingly